simulation_parameters/TranslationsInfo.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+public class TranslationsInfo : IRegistryType
+{
+    [JsonProperty]
+    public Dictionary<string, float> TranslationProgress { get; private set; }
+
+    /// <summary>
+    ///   Unused
```

Comment:
This seems auto-generated?

Reply:
Nope, copy-pasted from another system, this is needed because `IRegistryType` requires this.